repositories/apiserver.go

```diff
@@ -140,9 +156,13 @@ func (a *APIServerStore) GetSBOM(ctx context.Context, imageID, SBOMCreatorVersio
 }
 
 func (a *APIServerStore) GetSBOMp(ctx context.Context, instanceID, SBOMCreatorVersion string) (sbom domain.SBOM, err error) {
+	if instanceID == "" {
+		return domain.SBOM{}, nil
+	}
 	manifest, err := a.StorageClient.SBOMSPDXv2p3Filtereds(a.Namespace).Get(ctx, hashFromInstanceID(instanceID), metav1.GetOptions{})
 	if err != nil {
-		return domain.SBOM{}, err
+		logger.L().Ctx(ctx).Warning("failed to get relevant SBOM from apiserver", helpers.Error(err), helpers.String("ID", instanceID))
```

Comment:
What if the storage is not running? Will we still see this warning message?

Reply:
I think this will happen a lot because the relevant SBOM is ready only after X  minutes.
It should be a debug log or no log at all